JsonE/Operators/JsonOperator.cs

```diff
@@ -36,7 +39,8 @@ internal class JsonOperator : IOperator
 		{
 			dict[kvp.Key] = Sort(kvp.Value);
 		}
-
-		return JsonSerializer.SerializeToNode(dict);
+		
+		// Can't use the TypeInfo overload because JsonSerializerContext differs in nullability
+		return JsonSerializer.SerializeToNode(dict, JsonESerializerContext.Default.Options);
```

Comment:
Is the nullability an error or just a warning?

Reply:
It's this issue I filed:
```suggestion
		// Can't use the TypeInfo overload because JsonSerializerContext differs in nullability (https://github.com/dotnet/runtime/issues/97665)
		return JsonSerializer.SerializeToNode(dict, JsonESerializerContext.Default.Options);
```